db/schema/commonIssues.ts

```
@@ -0,0 +1,20 @@
+import { bigint, pgTable, text, timestamp } from "drizzle-orm/pg-core";
+import { withTimestamps } from "../lib/with-timestamps";
+import { mailboxes } from "@/db/schema/mailboxes";
+import { randomSlugField } from "../lib/random-slug-field";
+
+
+export const commonIssues = pgTable(
+    "common_issues", 
+    {
```

ESLint

[error] Replace `⏎····"common_issues",·⏎····` with `"common_issues",·`

(prettier/prettier)